### PR TITLE
Delay initial data loading (embedded JS in the HTML)

### PR DESCRIPTION
**Pull request by joakin**
Let's not duplicate the initial html size with the initial JS data and load it from a script tag instead.

### CODE REVIEW

File: lib/server/routes.js [hunk 1]
```diff
@@ -64,6 +53,40 @@ router.get('*', (req, res, next) => {
...
+    )
+    const state = JSON.stringify(store.getState())
+    res.status(200).send(
+      views.layout({
+        title: (renderProps.params.title || '') + ' - Wikipedia',
+        body: html,
+        url: req.url,
+        // Send inline data when it is smaller than 30kb. Else let the browser
+        // request it with a script tag
+        data: state.length < 30 * 1024 ? state : undefined
```
Comment (phuedx) [inline lib/server/routes.js]: It'd be nice to trigger this with a query parameter but this'll do for now. Also, 30 KB is going to be larger than most lead sections, so this – rather nice (!) – optimisation won't be invoked for most initial requests.

**Comment by phuedx** [comment]
If you were to make this configurable now, then we could compare the two approaches easily with WPT in a little while.

**Comment by joakin** [comment]
@phuedx  See 33ca014 ^

**Comment by phuedx** [comment]
pew pew

**Pull request merged**